artichoke-backend/src/types.rs

```diff
@@ -197,17 +200,23 @@ mod tests {
         assert_eq!(Ruby::Class, types::ruby_from_mrb_value(builtin.inner()));
         let data = interp.eval(b"Array").unwrap();
         assert_eq!(Ruby::Class, types::ruby_from_mrb_value(data.inner()));
-        let source = interp.eval(b"Math::DomainError").unwrap();
-        assert_eq!(Ruby::Class, types::ruby_from_mrb_value(source.inner()));
+        #[cfg(feature = "stlib-cmath")]
+        {
+            let source = interp.eval(b"Math::DomainError").unwrap();
+            assert_eq!(Ruby::Class, types::ruby_from_mrb_value(source.inner()));
+        }
     }
 
     #[test]
     fn parse_module_ruby_type() {
         let mut interp = interpreter().unwrap();
         let builtin = interp.eval(b"Comparable").unwrap();
         assert_eq!(Ruby::Module, types::ruby_from_mrb_value(builtin.inner()));
-        let data = interp.eval(b"Math").unwrap();
-        assert_eq!(Ruby::Module, types::ruby_from_mrb_value(data.inner()));
+        #[cfg(feature = "stlib-cmath")]
```

Comment:
I think this should be the `Math` core module:

```suggestion
        #[cfg(feature = "core-math")]
```